lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -104,6 +108,14 @@ def self.init_hypervisor(options)
         end
       end
 
+      # Verify that a valid hypervisor has been specified
+      # @param [Array<Object>] options the options we want to query
+      def self.verify_init_args(options)
```

Comment:
Yeahhh, when I started to add the provision helpers, I thought that it was starting to sprawl a bit. I guess as we add more subcommands that's only going to get worse. When you say sectioning, do you mean have an InitUtil module, ProvisionUtil module etc?